praw/models/reddit/subreddit.py

```diff
@@ -1664,6 +1664,22 @@ def subreddits(self):
             subreddit.last_updated = value['lastUpdated']
             yield subreddit
 
+    def unread_count(self):
+        """Return unread conversation count by conversation state.
+
+        :returns: A dict mapping conversation states to unred counts.
+
+        For example, to print the count of unread moderator discussions:
```

Comment:
Should we document the possible conversation states with a "at the time of this writing"?